### ISSUE TITLE
[Bug]: Changing default bottle path and Moving a bottle to new path clears all pinned programs from all bottles

### ISSUE THREAD

**Issue by WWWFWWW** [Whisky-App/Whisky#830]
### Description

Changing default bottle path and Moving a bottle there clears _all_ pinned programs from _all_ bottles, even the unmoved ones. Haven't checked if other configuration options persist or not.

### Steps to reproduce

1. have several bottles with pinned programs in them
2. change default bottle path
3. move a single bottle to the new default path using Whisky's Move function
4. all pinned programs from each and every bottle disappear

### Expected behaviour

pinned programs and other configuration options should persist

### Logs

```bash
Whisky Version: 2.2.3
Date: 2024-02-REDACTED
macOS Version: 14.3.1

Bottle Name: Bottle
Bottle URL: /Users/REDACTED/Library/Containers/com.isaacmarovitz.Whisky/Bottles/REDACTED

Wine Version: 7.7.0
Windows Version: win10
Enhanced Sync: msync

Metal HUD: false
Metal Trace: false

Arguments: -k

Environment:
{
    WINEDEBUG = "fixme-all";
    WINEPREFIX = "/Users/REDACTED/Library/Containers/com.isaacmarovitz.Whisky/Bottles/REDACTED";
}
```

### What version of Whisky are you using?

2.2.3

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by EmilianoTalamo** [comment]
Yup, just moved a bottle and it cleared all my pinned programs. Params and arguments are restored once you pin then back, but it's quite a hassle.
